CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 Changelog
 =========
 
-* **2019-06-18**: [BC BREAK] Removed all `*.class` parameters.
+* **2016-06-19**: [BC BREAK] Removed `setParent()`/`getParent()` from the model `MenuNode`.
```

Comment:
i think for better DX we should say something like "Removed the deprecated methods ... . Use setParentDocument / getParentDocument instead."